src/instructlab/eval/mmlu.py

```diff
@@ -79,7 +79,7 @@
 
 
 def run_mmlu(
-    model_path, model_dtype, tasks, few_shots, batch_size, sdg_path=None
+    model_path, model_dtype, tasks, few_shots, batch_size, device, sdg_path=None
```

Comment:
Can we make device optional?  Otherwise it's an api breaking change.

Reply:
I set a default in the cli https://github.com/instructlab/instructlab/pull/1672 but can change if we think it should be optional

Reply:
Recap from eval meeting today: I will change this PR to make 'device' optional and I will leave https://github.com/instructlab/instructlab/pull/1672 open with a hold for now